hive-metastore/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -251,12 +251,12 @@ public void createNamespace(Namespace namespace, Map<String, String> meta) {
             namespace);
 
     } catch (TException e) {
-      throw new RuntimeException("Failed to create namespace " + namespace + " in Hive MataStore", e);
+      throw new RuntimeException("Failed to create namespace " + namespace + " in Hive MetaStore", e);
```

Comment:
Out of curiosity, how do Hive folks prefer to write it? `MetaStore` or `Metastore`?

Reply:
I honestly always used Metastore and it's like that in most of the Hive docs, but I've seen the camel case a lot too.

+ @cwsteinbach  @pvary  ?

Reply:
I’ve also always used `Metastore`.

Looking at this Hive design doc, it’s also `Metastore` with no capital S. I think this makes the most sense as it’s not really two words imo (and several Hive docs confirm it as being one word): https://cwiki.apache.org/confluence/plugins/servlet/mobile?contentId=27362072#Design-Metastore

Reply:
I'd say Metastore. Capitalizing the S was to match the class name.